frontend/src/app/document-view/document-view.component.ts

```diff
@@ -71,6 +71,10 @@ export class DocumentViewComponent implements OnChanges {
         return field.name === 'url' || field.name.startsWith('url_');
     }
 
+    isGeoPointField(field: CorpusField) {
+        return field.name === 'coordinates';
```

Comment:
I don't think it's wise to hard-code the name as `coordinates` here, but rather check if the field has the [geopoint field type](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html) as `.es_mapping`.